build.gradle

```diff
@@ -134,6 +135,7 @@ javadoc {
     excludes = ['io/nats/client/impl', 'io/nats/examples']
     classpath = sourceSets.main.runtimeClasspath
     doLast {
+        if (!OperatingSystem.current().isWindows()) {
```

Comment:
this allows the build to work on windows while not affecting any other operating system